chore(docs): warn about features on Preview Branches (#23214)
supabase/supabase: apps/docs/content/guides/platform/branching.mdx

You can run multiple Preview Branches for every Supabase project. Branches contain all the Supabase features with their own API credentials. Preview Environments pause automatically after 5 minutes of inactivity.

<Admonition type="caution" label="Some features may not work as expected on Preview Branches">

> Due to Preview Branches running on Fly Postgres instances, some features may not work as expected. Most notably `pg-cron`.

Reviewer: ```suggestion
Due to Preview Branches running on Fly Postgres instances, some features like `pg-cron` are not fully supported.
```

> For a full list of features that may not work as expected, see the [Fly Postgres limitations](/docs/guides/platform/fly-postgres#limitations).

Reviewer: ```suggestion
For a full list of limitations, see [Fly Postgres limitations](/docs/guides/platform/fly-postgres#limitations).
```
